x-pack/plugin/security/build.gradle

```diff
@@ -79,7 +79,7 @@ dependencies {
   runtimeOnly "joda-time:joda-time:2.10.10"
 
   // Dependencies for oidc
-  api "com.nimbusds:oauth2-oidc-sdk:11.10.1"
+  api "com.nimbusds:oauth2-oidc-sdk:11.22.2"
```

Comment:
I've only checked transitive dependencies of the `json-smart`, but we should bump all transitive dependencies of `com.nimbusds:oauth2-oidc-sdk` as well.